templates/master.yaml

```diff
@@ -576,39 +469,35 @@ coreos:
 
       [Install]
       WantedBy=multi-user.target
-  # TODO(nhlfr): Set up Calico on Kubernetes, in example by http://docs.projectcalico.org/v2.0/getting-started/kubernetes/installation/hosted/kubeadm/calico.yaml.
-  # Or at least use anything which doesn't download binaries in systemd unit...
```

Comment:
This TODO is pretty valid. We should only remove it when it is addressed. There should also be an issue for this. Feel free to work on this topic next. For now I think we should keep it.